### PR TITLE
[#97] Adds docs about vk data attributes

### PR DESCRIPTION
**Pull request by vitkarpov**

### CODE REVIEW

**Comment by vitkarpov** [comment]
bummer! facebook counters fail the tests :)

File: readme.md [hunk 1]
```diff
@@ -148,6 +148,10 @@ The attribute should be an image URL:
 <div class="pinterest" data-media="https://placekitten.com/200/400">Pin</div>
 ```
 
+### VK
```
Comment (iamakulov) [inline readme.md]: This is in the middle of the “Pinterest” section :–)
Let’s place this section after Twitter, OK? VK is more important than Telegram or Pinterest but less important than Twitter.

Comment (iamakulov) [inline readme.md]: Also, could you please add an example?

**Comment by iamakulov** [comment]
> bummer! facebook counters fail the tests :)

Weird, restarted the build.

**Comment by iamakulov** [comment]
Up?

**Comment by iamakulov** [comment]
@vitkarpov Thanks 🙇

**Pull request merged**